src/DiIiS-NA/D3-GameServer/GSSystem/PowerSystem/Implementations/HeroSkills/Wizard.cs

```diff
@@ -3168,14 +3174,15 @@ public override IEnumerable<TickTimer> Main()
 								}
 					}
 
-				var projectiles_around = TimeWarp.GetObjectsInRange<Projectile>(40f).Where(p => PowerMath.Distance2D(p.Position, TimeWarp.Position) > 20f);
-				if (projectiles_around.Count() > 0)
```

Comment:
Here we filtered twice - by `Count()` and then by foreach.
And so on and so forth